src/celeritas/ext/GeantImporter.cc

```diff
@@ -192,6 +192,72 @@ struct MatPropGetter
     }
 };
 
+//---------------------------------------------------------------------------//
+//! Map particles defined in \c G4MaterialConstPropertyIndex .
+std::map<std::string, PDGNumber> optical_particles_map()
+{
+    return {{"PROTON", pdg::proton()},
```

Comment:
This should be a `static const` so it's created once and reused.

```suggestion
auto& optical_particles_map()
{
    static std::unordered_map<std::string, PDGNumber> const {{"PROTON", pdg::proton()},
```